static/include/app/apis/generated/data.md

```diff
@@ -350,16 +382,21 @@ Add tags to binary data by ids.
 
 ```python {class="line-numbers linkable-line-numbers"}
 from viam.proto.app.data import BinaryID
+from viam.utils import create_filter
 
 tags = ["tag1", "tag2"]
 
+my_filter = create_filter(component_name="camera-1", organization_ids=["<YOUR-ORG-ID>"])
 binary_metadata = await data_client.binary_data_by_filter(
```

Comment:
```suggestion
binary_metadata, count, last = await data_client.binary_data_by_filter(
```

Reply:
see above but this throws an error for me